src/Uno.UWP/Storage/FileIO.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Globalization;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Uno.Extensions;
+using Windows.Foundation;
+using Windows.Storage.FileProperties;
+using Windows.Storage.Streams;
+
+namespace Windows.Storage
+{
+	public partial class FileIO
+
+		public static global::Windows.Foundation.IAsyncAction WriteTextAsync( global::Windows.Storage.IStorageFile file,  string contents)
+		{
+        Stream fileStream = await file.OpenStreamForWriteAsync();
+        StreamWriter streamWriter = new StreamWriter(fileStream);
+        streamWriter.Write(contents); 
+        streamWriter.Dispose();
+        fileStream.Dispose();
```

Comment:
I would suggest using `using` blocks for the `StreamWriter` and `FileStream` instead. In case an exception happens, in the current implementation the file could stay in open state.

Reply:
I don't know architecture of tests. E.g. I don't know if every test has own filesystem. Or, can file from one test be seen by other tests?

Reply:
You should assume that a test _might_ see files from other tests (you can't know for sure because you don't know which order tests will run in). So you should clean up files after each test (probably in a tear-down method).

You should probably be writing runtime unit tests for this: https://platform.uno/docs/articles/contributing/guidelines/creating-tests.html#platform-runtime-unit-tests-unouiruntimetests

Reply:
Ideally you should perform cleanup after the test is finished - so wrapping the unit test in a `try...finally` and cleaning up in finally. Each build has its own separate file store, but still, better safe than sorry :-D . As a sample, you can check [this unit test](https://github.com/unoplatform/uno/blob/024eebddd33ac0dfa6b6d8ea0871d5c6effc9f12/src/Uno.UI.Tests/Windows_UI_Xaml/Given_ThemeResource.cs#L31 ) for resources - I am using finally there to reset back to defaults. Tests for this API would probably fit best in the `Uno.UI.RuntimeTests` library.